change/@ni-nimble-components-0d465c53-f139-410c-b252-e3da15e453e2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: when fixing issues that don't require validation (like tech debt or small bugs) we can use github keywords in the PR description to automatically close them (e.g. "fixes #xxx").